server/enterprise/message_export/worker.go

```diff
@@ -362,6 +366,35 @@ func (w *MessageExportWorker) initJobData(logger mlog.LoggerIFace, job *model.Jo
 	job.Data[shared.JobDataExportDir] = getJobExportDir(logger, job.Data, job.Data[shared.JobDataJobStartTime], job.Data[shared.JobDataJobEndTime])
 }
 
+func (w *MessageExportWorker) getPreviousJob(rctx request.CTX) (*model.Job, error) {
```

Comment:
This is not exactly previous job right? This is just returning the first non-mmctl initiated export job. In case of multiple CLI job instantiations, this method name is inaccurate. Can we rename it to something better?

Reply:
Good idea, fixed.